test/snoowrap.spec.js

```diff
@@ -334,30 +334,33 @@ describe('snoowrap', function () {
       expect(snoowrap.objects.Comment).to.exist();
     });
     it('exposes camelCase aliases for all prototype functions', () => {
+      // noinspection JSUnresolvedVariable
```

Comment:
Can you remove these `noinspection` comments? Otherwise this looks good to me.

Reply:
Fixed :)